pull/context.go

```diff
@@ -65,6 +65,12 @@ type Context interface {
 	// CreatedAt returns the time when the pull request was created.
 	CreatedAt() time.Time
 
+	// IsOpen returns true when the state of the pull request is "open"
+	IsOpen() bool
+
+	// IsClosed returns true when the state of the pull request is "closed"
+	IsClosed() bool
+
```

Comment:
Perhaps `IsClosed` is not necessary yet, but it felt right to add since `open` and `closed` are the two possible values (right now).